### ISSUE TITLE
Black color is based on background color?

### ISSUE THREAD

**Issue by nirewen** [zeit/hyper#3302]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Windows 10 (Build 18277.rs_prerelease.181102-1446)
- **Hyper.app version**: 2.0.0 (stable)
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/nirewen/0cee955353b586ac567e8aa5c1b95894
- **The issue is reproducible in vanilla Hyper.app**: Yes

## Issue

Today I discovered this beautiful app called Hyper, and installed it.
I was messing around menus and I found that I can edit the colors of the app, and so I did.

The issue is: The background color and `black` are set to different colors (`#1C1C1C` and `#000000`), though, the black color seems to be the same as the background color (see attached screenshot)

I don't think this is an issue with chalk, but maybe an issue with my native terminal (?)
Do the colors of the native terminal interfere in Hyper?

My .hyper.js configuration is above, if someone spot my mistake, let me know.

![image](https://user-images.githubusercontent.com/8761479/49315371-9f709600-f4d4-11e8-8b0b-b391f74affec.png)

**Comment by robertgrzonka** [comment]
Well, the difference between this two `#000000` and `#1C1C1C` is really small. I may be wrong but I guess that your Windows (as default) supports `256 colors`. This two shades of black are included as one color and this is why you see them as one color: `black`. Google them and check if you see the difference between them when set next to each other. What are you interested in is called `true colors` which contains ~16 million colors.

**Comment by nirewen** [comment]
The colors are not that similar at all. Even though, if they were indeed the same color, I could still change the background to another color and `#000000` would show, right?
Here are the colors side by side
![image](https://user-images.githubusercontent.com/8761479/49340561-a3adc800-f628-11e8-9743-4ac1abbc4723.png)

Also, I tried resetting the terminal as https://github.com/zeit/hyper/issues/1706#issuecomment-302873966 suggested, didn't fix my problem

**Comment by nirewen** [comment]
Doing some research, I found this: [True Colour (16 million colours)](https://gist.github.com/XVilka/8346728#not-supporting-truecolour)
Hyper.app is listed in the NOT Supporting section.
So in its current version 2.0.0, it doesn't support True Color

**Comment by robertgrzonka** [comment]
Yes, correct: `Hyper` is using `xterm-256colors` protocole which u can check typing in your shell as below:
```zsh
echo $TERM    #should output: `xterm-256color`
```
![zrzut ekranu 2018-12-2 o 20 55 28](https://user-images.githubusercontent.com/35585466/49344236-a59d7880-f674-11e8-8888-a845e26eabce.png)
Please check this gray color `#404040` if it suites your needs. [This](https://en.wikipedia.org/wiki/X11_color_names) may be helpful when looking for another solutions.
